packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -89,6 +92,20 @@ export class BlockList extends Component {
 		return renderAppender && blockClientIds.length > 0;
 	}
 
+	getVerticalAlignmentRemap( alingment ) {
```

Comment:
I think this implementation could be a bit simpler. Also there's a typo in the argument and I think `getVerticalAlignmentStyle` might be a better name:

```jsx
getVerticalAlignmentStyle( alignment ) {
	const mapping = {
		top: 'flex-start',
		center: 'center',
		bottom: 'flex-end',
	};

	const justifyContent = mapping[ alignment ];
	return { justifyContent }
}
```

Reply:
how about `mapAlignmentPropValue` ? and below snippet (assuming that we can also extract `mapping` outside the function

```
	getVerticalAlignmentRemap( alignment ) {
		const mapping = {
			top: 'flex-start',
			center: 'center',
			bottom: 'flex-end',
		};
		const newAlignment = get( mapping, alignment );
		return newAlignment ? { justifyContent: newAlignment } : undefined;
	}
```

Reply:
done!